lensless/unrolled_admm.py

```diff
@@ -83,6 +87,10 @@ def __init__(
         self._mu3_p = torch.nn.Parameter(torch.ones(self._n_iter, device=self._psf.device) * mu3)
         self._tau_p = torch.nn.Parameter(torch.ones(self._n_iter, device=self._psf.device) * tau)
 
+        self.post_process = post_process
+        if self.post_process is not None:
+            self.noise_level = torch.nn.Parameter(torch.tensor([1.0], device=self._psf.device))
```

Comment:
given you assume a denoiser, should we rename "post_process" to "denoiser"?

Reply:
also it's rather hidden, but the denoiser parameters are also finetuned right?

Reply:
and if so, is there a flag that can be set so that the denoiser is not fine-tuned?